server/src/app.js

```diff
@@ -20,12 +21,22 @@ const passport = require('passport')
 const RedisStore = require('connect-redis')(session)
 const redis = require('./redis')
 const elasticsearch = require('./elasticsearch')
+const Raven = require('raven')
 const startSubscriptionServer = require('./graph/subscriptions/SubscriptionServer')
 const PORT = 8181
 
+const GIT_HASH = execSync('git rev-parse HEAD', {
```

Comment:
Running a synchronous shell command at server spinup feels icky to me. Not sure what your prod setup looks like, but a better approach might be to build a docker image for every backend deploy and embed the git commit during the image build step. Could also include the docker image hash in that case (possibly w/ Sentry's release tracking functionality).